Apps/Laboratory/src/Factories/LaboratoryDelegateFactory.cs

```diff
@@ -16,13 +16,15 @@
 namespace HealthGateway.Laboratory.Factories
 {
     using System;
+    using System.Diagnostics.CodeAnalysis;
     using HealthGateway.Laboratory.Delegates;
     using Microsoft.Extensions.Configuration;
     using Microsoft.Extensions.DependencyInjection;
 
     /// <summary>
     /// Creates the Laboratory Delegate dynamically.
     /// </summary>
+    [ExcludeFromCodeCoverage]
```

Comment:
Same comment as above, can't we test this?

Reply:
I can try as it includes a MS lib. I was wondering if testing this class brings us a lot of value.

Reply:
Agree with laws, this is part of the production code, so it should be tested

Reply:
updated